tensorflow/python/kernel_tests/signal/fft_ops_test.py

```diff
@@ -662,6 +662,13 @@ def test_placeholder(self, axes):
     self.assertAllClose(y_fftshift_res, np.fft.fftshift(x_np, axes=axes))
     self.assertAllClose(y_ifftshift_res, np.fft.ifftshift(x_np, axes=axes))
 
+  def test_negative_axes(self):
+    with self.session():
+      freqs = [[0, 1, 2], [3, 4, -4], [-3, -2, -1]]
+      shifted = [[-1, -3, -2], [2, 0, 1], [-4, 3, 4]]
+      self.assertAllEqual(fft_ops.fftshift(freqs, axes=(0, -1)), shifted)
```

Comment:
Can you test for negative scalar axes too (like axes=-1)?

Reply:
@alextp Thanks. The PR has been updated with additional test cases added.